bin/ecs-task-runner

```diff
@@ -81,7 +79,7 @@ const options = {
   securityGroups: argv.securityGroups
 };
 
-ecsTaskRunner(options, function(err, stream) {
+ecsTaskRunner(options, function (err, stream) {
```

Comment:
Most (all?) of these are due to my editor auto-formatting. Apologies.

Reply:
That's like apologizing for your dog barking when a stranger is poking around your windows at night, it's doing its job and doing it well :)

Reply:
I may very well have different settings though, I don't think we've got any editor settings file in this repo (and I'm too lazy to create one right now).